src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -92,6 +94,12 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
   const handleDeleteOpen = () => setDeleteModalShow(true);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const dropdownOpen = Boolean(anchorEl);
+  const { data: project } = useSingleProject({
+    carNumber: changeRequest.wbsNum.carNumber,
+    projectNumber: changeRequest.wbsNum.projectNumber,
+    workPackageNumber: 0
+  });
+  const { name: projectName } = project || {};
```

Comment:
I think that { name: projectName } = project || {} b/c project can be undefined. I think this line works fine, but wondered if there's any better way to specify { name: projectName }, so the code is easier to read. Let me know from anyone who reviews this PR if this is a necessary thing we need to improve and if so, how. Thanks!